### PR TITLE
:fire: Remove explicit tokens

### PR DESCRIPTION
**Pull request by flaxel**
### Type of Change

- [ ] Bugfix
- [ ] Enhancement / new feature
- [x] Refactoring
- [ ] Documentation

### Description

- remove explicit tokens
- usage of one of the docker build secrets properties is recommended

### Checklist

- [x] Add relevant labels (for example type of change or patch/minor/major)
- [x] Make sure not to introduce some mistakes
- [x] Update documentation
- [x] Review the [Contributing Guideline](../blob/main/CONTRIBUTING.md) and sign CLA
- [x] Reference relevant issue(s) and close them after merging

### CODE REVIEW

File: .github/workflows/template_gitops.yml [hunk 1]
```diff
@@ -61,18 +56,13 @@ jobs:
         with:
           docker-username: ${{ secrets.docker-username }}
           docker-password: ${{ secrets.docker-password }}
-          # remove npm token, goproxy, gonosumdb if feature is available: https://github.com/github-community/community/discussions/17554
-          docker-build-args: |
-            ${{ inputs.docker-build-args }}
-            NPM_TOKEN=${{ secrets.npm-token }}
-            GOPROXY=${{ secrets.goproxy }}
-            GONOSUMDB=${{ secrets.gonosumdb }}
```
Comment (staust) [inline .github/workflows/template_gitops.yml:69]: I'm not sure about the gonosumdb ... basically this is no secret and for me it is still okay to use this as build argument. I guess we should still allow that

Comment (flaxel) [reply .github/workflows/template_gitops.yml:69]: I would maybe leave it as we defined it as company secret and then maybe that's a little more consistent? 🤔

**Review by timdittler — CHANGES_REQUESTED** [review]
I'm missing a piece of advice on how to migrate from explicit tokens to docker build secrets. Would be great if not every team would have to figure this out by themselves.

**Comment by flaxel** [comment]
@timdittler Yeah sure, but I would add this information directly to the release. I would write something like that: 

```
Breaking Changes:
If you are using one of these variables ..., you need to adapt your workflow and move these variables to the parameter.
```

Is this okay? 🤔

**Comment by staust** [comment]
> I'm missing a piece of advice on how to migrate from explicit tokens to docker build secrets. Would be great if not every team would have to figure this out by themselves.

here is an example: https://github.com/Staffbase/play-store-connect-service/pull/92/files

**Review by timdittler — APPROVED** [review]

**Review by staust — APPROVED** [review]

**Pull request merged**